docs-v2/content/en/schemas/version-mappings/v4beta10-version.json

```diff
@@ -0,0 +1 @@
+{"binVersion":"2.11.0","releaseNoteLink":"https://github.com/GoogleContainerTools/skaffold/releases/tag/2.11.0"}
```

Comment:
The release script is not generating the `releaseNoteLink` properly under some parameters, for this case the last part of the link should be `/v2.11.0` instead of `/2.11.0`, could you change it manually? We'll have to fix the release script to not have this problem in future releases. Thanks!

Reply:
ahhh, thanks